parser-typechecker/src/Unison/Util/Bytes.hs

```diff
@@ -35,12 +38,27 @@ empty = Bytes mempty
 fromArray :: B.ByteArrayAccess ba => ba -> Bytes
 fromArray = snoc empty
 
+zlibCompress :: Bytes -> Bytes
+zlibCompress = fromLazyByteString . Zlib.compress . toLazyByteString
+
+gzipCompress :: Bytes -> Bytes
+gzipCompress = fromLazyByteString . GZip.compress . toLazyByteString
+
+gzipDecompress :: Bytes -> Bytes
+gzipDecompress = fromLazyByteString . GZip.decompress . toLazyByteString
+
+zlibDecompress :: Bytes -> Bytes
+zlibDecompress = fromLazyByteString . Zlib.decompress . toLazyByteString
+
```

Comment:
Consider putting these into `Unison.Util.Bytes.Compress` or something, though I don't feel super strongly in this case.